### PR TITLE
Module 1 Portfolio desktop version

### PR DESCRIPTION
**Pull request by the-catalystmc**
In this project the following was done:

- Parse a design from FIgma for the desktop version of Portfolio.
- Use media queries to make sure that the website displays correctly on desktop screens.
- Add a fixed Navbar
- Make sure that there are no linter errors.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -412,3 +448,230 @@ ul .small-button-wrapper {
   margin-bottom: 0;
   font-size: 15px;
 }
+
+@media (min-width: 992px) {
```
Comment (vichuge) [inline style.css]: - [x] Please, use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px), for example: `@media screen and (min-width: 992px) {` 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @the-catalystmc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters errors ✔️ 
- Good desktop view ✔️ 

## Aspects to improve ♻️

- [x] Maybe the style for diff sizes in responsible could improve, look how appears your page on diff sizes (I recommend you use Chrome dev tools to check the diff sizes for your page)

| Desktop | Ipad |
| --- | --- |
| image | image |

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL] Use more details on form in every sizes
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -412,3 +448,230 @@ ul .small-button-wrapper {
   margin-bottom: 0;
   font-size: 15px;
 }
+
+@media screen and (min-width: 992px) {
```
Comment (Davidosky007) [inline style.css]: I suggest you increase the min-width as the previous reviewer asked to eg (min-width: 993 - upward) min-width should be greater than 992px to 1100px as this is the required media query for desktop applications

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @the-catalystmc, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 2 above)
Comment (Davidosky007) [reply style.css]: nice job 👍

**Comment by Davidosky007** [comment]
Hi @the-catalystmc,
CR GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**